.gitignore

```diff
@@ -79,4 +79,5 @@ dist
 .vscode
 
 # Heft
+.cache
```

Comment:
Should this folder be called `.heft-cache`?

Reply:
I'm indifferent. I kinda like `.cache` because it's shorter and more generic. There's nothing stopping someone else from using it anyway, and doing so generally shouldn't interfere with our use since we use it to store per-task caching subfolders (generated from phase/task names).